cmd/polaris/audit.go

```diff
@@ -77,6 +86,32 @@ var auditCmd = &cobra.Command{
 	},
 }
 
+// ProcessHelmTemplates turns helm into yaml to be processed by Polaris or the other tools.
+func ProcessHelmTemplates(helmChart string) (string, error) {
+	cmd := exec.Command("helm", "dependency", "update", helmChart)
+	err := cmd.Run()
+	if err != nil {
+		return "", err
+	}
+
+	dir, err := ioutil.TempDir("", "*")
+	if err != nil {
+		return "", err
+	}
+	params := []string{
+		"template", helmChart,
+		helmChart,
+		"--output-dir",
+		dir,
+	}
+	cmd = exec.Command("helm", params...)
+	err = cmd.Run()
+	if err != nil {
```

Comment:
It looks like the `err` here has some extra info that doesn't get logged automatically when we `panic`. Let's add some more logging to show the `stderr` output when the command fails.

For  background, [here are the docs on cmd.Run()](https://golang.org/pkg/os/exec/#Cmd.Run). You can see there that sometimes the `error` it returns is an [ExitError](https://golang.org/pkg/os/exec/#ExitError)

What we need to do is:
* check if this is an ExitError
* if so, log stderr

tl;dr - I think we want to add this code:
```go
if err != nil {
  if exitErr, ok := err.(exec.ExitErrror); ok {
    logrus.Error(string(exitErr.Stderr))
  }
  return "", err
}
```

to test it out, you can try passing in `--helm-chart=/a/path/that/doesnt/exist`